ui/js/services/perf/math.js

```diff
@@ -1,4 +1,4 @@
-import _ from 'lodash';
+
```

Comment:
Could you also remove the newlines at the start of this file? :-)

Reply:
(In addition to the change here: https://github.com/mozilla/treeherder/pull/3675#discussion_r195943456)